dev-tools/pom.xml

```diff
@@ -0,0 +1,29 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.elasticsearch</groupId>
+  <artifactId>dev-tools</artifactId>
+  <version>2.0.0-SNAPSHOT</version>
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-remote-resources-plugin</artifactId>
```

Comment:
Forget about that, you would need to add the parent-pom here, too, but that would produce a cycle or many additional executions.

But you should add the groupId:

```xml
<groupId>org.apache.maven.plugins</groupId>
<artifactId>maven-remote-resources-plugin</artifactId>
```